nilearn/maskers/nifti_labels_masker.py

```diff
@@ -111,6 +116,33 @@ class NiftiLabelsMasker(BaseMasker, _utils.CacheMixin):
 
         .. versionadded:: 0.9.2
 
+    region_ids_ : dict
+        A dictionary containing the region ids corresponding
+        to each column in region_signal.
+        The region id corresponding to ``region_signal[:,i]``
+        is ``region_ids_[i]``.
+        ``region_ids_['background']`` is the background label.
+
+        .. versionadded:: 0.11.0.dev
+
+    region_names_ : dict
+        A dictionary containing the region names corresponding
+        to each column in region_signal.
+        The region names correspond to the labels provided
+        in labels in input.
+        The region name corresponding to ``region_signal[:,i]``
+        is ``region_names_[i]``.
+
+        .. versionadded:: 0.11.0.dev
+
+    region_atlas_ : Niimg-like object
+        Regions definition as labels.
+        The labels correspond to the indices in ``region_ids_``.
+        The region in ``region_atlas_`` that takes the value ``region_ids_[i]``
+        is used to compute the signal in ``region_signal[:,i]``.
+
+        .. versionadded:: 0.11.0.dev
+
```

Comment:
@mtorabi59 
Random question on this PR: why was it chosen for those attributes to have a trailing underscore?

Why couldn't we have:
- region_names
- region_atlas
- region_ids

Reply:
I think it means that these quantities are estimated, i.e. fit-/data-dependent.

Reply:
ha. I guess we should probably mention that somewhere in our style guide, because it is clearly not obvious.

Reply:
I am fine just pointing to the scikit learn doc: https://scikit-learn.org/stable/developers/develop.html#estimated-attributes